Support passing source and workspace flag (#989)
kyma-project/cli: docs/gen-docs/kyma_alpha_deploy.md

  -c, --components-file string   Path to the components file (default "$HOME/.kyma/sources/installation/resources/components.yaml" or ".kyma-sources/installation/resources/components.yaml")
  -d, --domain string            Custom domain used for installation.
  -p, --profile string           Kyma deployment profile. If not specified, Kyma uses its default configuration. The supported profiles are: evaluation, production.
  -s, --source string            Installation source:
                                 	- Deploy a specific release, for example: "kyma deploy --source=2.0.0"
                                 	- Deploy a specific branch of the Kyma repository on kyma-project.org: "kyma deploy --source=<my-branch-name>"

>                                  	- Deploy a commit (should be 8 characters long), for example: "kyma deploy --source=34edf09a"

Reviewer: ```suggestion
                                 	- Deploy a commit (8 characters or more), for example: "kyma deploy --source=34edf09a"
```

Reviewer 2: fixed
